cmd/kube-controller-manager/app/core.go

```diff
@@ -118,13 +120,32 @@ func startNodeIpamController(ctx ControllerContext) (http.Handler, bool, error)
 		}
 	}
 
+	nodeCIDRMaskSizes, err := processNodeCIDRMasks(ctx.ComponentConfig.NodeIPAMController.NodeCIDRMaskSize)
+	if err != nil {
+		return nil, false, err
+	}
+	// failure: length of node cidr masks greater than length of cluster cidrs
+	if len(nodeCIDRMaskSizes) > len(clusterCIDRs) {
+		return nil, false, fmt.Errorf("len of node cidr masks is %v > len of cluster cidrs", len(nodeCIDRMaskSizes))
+	}
+	// populate default mask size for each cluster cidr that doesn't have mask defined
+	if len(nodeCIDRMaskSizes) < len(clusterCIDRs) {
+		for _, clusterCIDR := range clusterCIDRs[len(nodeCIDRMaskSizes):] {
+			if netutils.IsIPv6CIDR(clusterCIDR) {
+				nodeCIDRMaskSizes = append(nodeCIDRMaskSizes, 98)
```

Comment:
Due to this limitation - https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/nodeipam/ipam/cidrset/cidr_set.go#L43-L46, the cluster cidr defined for ipv6 should have a mask >=82

Reply:
done!